lib/experiments/index.js

```diff
@@ -3,6 +3,7 @@ var symbol = require('../utilities/symbol');
 
 var experiments = {
   BUILD_INSTRUMENTATION: symbol('build-instrumentation'),
```

Comment:
I think the plan for now is to check for `BUILD_INSTRUMENTATION` for a bit and hard error, telling the user to move to `INSTRUMENTATION`.  Later yeah definitely remove it from experiments entirely.